### PR TITLE
Create strategic-entrepreneurship-journal.csl

### PR DESCRIPTION
**Pull request by fmotoki**
Modified the Strategic Management Journal style to comply with SEJ guidelines

### CODE REVIEW

File: strategic-entrepreneurship-journal.csl [hunk 1]
```diff
@@ -0,0 +1,237 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Strategic Entrepreneurship Journal</title>
+    <id>http://www.zotero.org/styles/strategic-entrepreneurship-journal</id>
+    <link href="http://www.zotero.org/styles/strategic-entrepreneurship-journal" rel="self"/>
```
Comment (POBrien333) [inline strategic-entrepreneurship-journal.csl]: Please include a link to the style you used as a template.

Comment (POBrien333) [inline strategic-entrepreneurship-journal.csl]: Please include a link to the style you used as a template.

File: strategic-entrepreneurship-journal.csl [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    <link href="https://onlinelibrary.wiley.com/page/journal/1932443x/homepage/forauthors.html" rel="documentation"/>
+    <author>
+      <name>Aku Valtakoski</name>
+      <email>aku.valtakoski@tkk.fi</email>
+    </author>
+    <contributor>
+      <name>Fabio Motoki</name>
```
Comment (POBrien333) [inline strategic-entrepreneurship-journal.csl]: Please make yourself the author (by removing Aku). Having the template link already includes Aku's work.

File: strategic-entrepreneurship-journal.csl [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    </contributor>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <issn>1932-443X</issn>
```
Comment (POBrien333) [inline strategic-entrepreneurship-journal.csl]: ISSN: 1932-4391
E-ISSN: 1932-443X
Please add the ISSN and relabel the one you have with eissn.

**Comment by rmzelle** [comment]
Thanks both!

(@POBrien333, you might want to give "Suggested Changes" a try. See https://blog.github.com/changelog/2018-10-16-suggested-changes/. When you comment on a line in a pull request there is an icon to make a code change suggestion.)

**Comment by POBrien333** [comment]
Ah cool. Wasn't aware of that. Thanks :)

**Comment by rmzelle** [comment]
@POBrien333, I don't know if I can accept such changes as repo maintainer if you make them, but it's worth a try.

**Pull request merged**